tests/js/object-prop-break-in/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -36,8 +36,53 @@ const x = {
+var obj = {
+  // an entry with a very long string
+  x: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  url: 'http://example.com/12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890',
+  longName: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  [i]: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  [prop]: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  'x': "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  a: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  ab: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  abc: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  abcd: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  abcde: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  abcdef: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+};
+
+=====================================output=====================================
+var obj = {
+  // an entry with a very long string
+  x: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  url: "http://example.com/12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  longName:
+    "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  [i]: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  [prop]:
+    "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  x: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  a: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  ab: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  abc: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
+  abcd: "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
```

Comment:
```
'abcd': "12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
```

Should be same?

Reply:
I don't think so.

Reply:
The overall length of the part before `:` is measured.
```
abcd → 4
'abcd' → 6
[x] → 3
```